docs/visualization/visualize_unit_metrics.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
<p>can be "found" there</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>